src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -56,12 +56,16 @@ class LocalBackend extends Backend with LazyLogging {
     val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr", callDirectory)
     val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command", callDirectory)
 
-    commandWriter.write(commandLine)
-    commandWriter.flushAndClose()
-
     val hostWorkflowExecutionFile = hostExecutionPath(workflowDescriptor).toFile
     val containerExecutionDir = s"/root/${workflowDescriptor.id.toString}"
 
+    if (call.docker.isEmpty) {
```

Comment:
meta-comment, it annoys me that we can say `foreach` on an option but need an explicit branch for None. Silly scala